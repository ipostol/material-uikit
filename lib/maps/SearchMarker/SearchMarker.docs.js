'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _reactMarkupComponents = require('react-markup-components');

var _SearchMarker = require('./SearchMarker');

var _SearchMarker2 = _interopRequireDefault(_SearchMarker);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = function () {
  return _react2.default.createElement(
    _reactMarkupComponents.Skin,
    null,
    _react2.default.createElement(
      'div',
      { style: { position: 'relative' } },
      _react2.default.createElement(_SearchMarker2.default, null)
    )
  );
};