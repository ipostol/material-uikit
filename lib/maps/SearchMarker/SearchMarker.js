'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _SearchMarker = require('./SearchMarker.scss');

var _SearchMarker2 = _interopRequireDefault(_SearchMarker);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var pinSearchImg = require('./images/pinSearch.svg');

var SearchMarker = function SearchMarker() {
  return _react2.default.createElement('img', { className: _SearchMarker2.default.general, src: pinSearchImg, alt: 'Search Marker' });
};

exports.default = SearchMarker;