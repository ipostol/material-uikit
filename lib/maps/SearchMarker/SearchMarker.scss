@import '~bidstack-theme/theme';

.general {
  width: 70px;
  height: 70px;
  position: absolute;
  top: -25px;
  left: -15px;
}
