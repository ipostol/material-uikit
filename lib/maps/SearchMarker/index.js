'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = undefined;

var _SearchMarker = require('./SearchMarker');

var _SearchMarker2 = _interopRequireDefault(_SearchMarker);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = _SearchMarker2.default;