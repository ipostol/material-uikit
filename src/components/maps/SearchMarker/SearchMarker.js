import React from 'react';
import styles from './SearchMarker.scss';

const pinSearchImg = require('./images/pinSearch.svg');

const SearchMarker = () => (
  <img className={styles.general} src={pinSearchImg} alt="Search Marker" />
);

export default SearchMarker;
