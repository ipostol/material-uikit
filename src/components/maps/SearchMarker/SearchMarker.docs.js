import React from 'react';
import { Skin } from 'react-markup-components';
import SearchMarker from './SearchMarker';

export default () => (
  <Skin>
    <div style={{ position: 'relative' }}>
      <SearchMarker />
    </div>
  </Skin>
);
