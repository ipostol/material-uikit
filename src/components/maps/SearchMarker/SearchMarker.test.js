import React from 'react';
import { shallow } from 'enzyme';
import SearchMarker from './SearchMarker';

describe('SearchMarker component', () => {

  it('should render correctly', () => {

    const props = {};

    const wrapper = shallow(
      <SearchMarker {...props} />,
    );

    expect(wrapper).toMatchSnapshot();

  });

});
