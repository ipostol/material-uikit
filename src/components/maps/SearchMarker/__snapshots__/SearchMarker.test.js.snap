// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchMarker component should render correctly 1`] = `
<img
  alt="Search Marker"
  className="general"
  src={Object {}}
/>
`;
